src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started to work");

```

Comment:
The message "Truck started to work" should be more informative and indicate that it is a Truck that is working. Consider specifying the action the Truck is performing.